### PR TITLE
Added TV-Shows section

### PR DESCRIPTION
**Pull request by iMoose**

### CODE REVIEW

**Comment by mstuttgart** [comment]
HI @iMoose thanks to you PR. Please rebase you branch to the latest commit.

Let me know if you need help.

**Comment by iMoose** [comment]
Conflict resolved

**Review by mstuttgart — CHANGES_REQUESTED** [review]
Thanks to resolve the conflict, but the README file is wrong. 

The `book` section is in the wrong place, I believe than rebase is wrong.

**Comment by mstuttgart** [comment]
You can rebase you repo like the:

```
git remote add upstream git@github.com:mstuttgart/awesome-time-travel.git
git fetch upstream
git pull --rebase upstream master
``` 
Fix the conflicts add tip:

```
git add .
git rebase --continue
git push -f origin patch-2
```

File: README.md [hunk 1]
```diff
@@ -21,7 +22,6 @@ A collection of anime and animations about time travel.
 
 A collection of scientific and not scientific books about time travel.
 
-* [11/22/63](https://en.wikipedia.org/wiki/11/22/63) - Stephen King - 2011
```
Comment (mstuttgart) [inline README.md]: @iMoose this movie was removed. Please fix this

**Review by mstuttgart — CHANGES_REQUESTED** [review]

**Comment by mstuttgart** [comment]
@iMoose thanks for this PR, but the `TV-show` section already added.

**Pull request closed without merging**